services/spar/test-integration/Util/Scim.hs

```diff
@@ -117,26 +119,30 @@ randomScimUserWithSubjectAndRichInfo ::
   m (Scim.User.User SparTag, SAML.UnqualifiedNameID)
 randomScimUserWithSubjectAndRichInfo richInfo = do
```

Comment:
there is a duplicate of this in brig that may also need updating?

also, could you add a FUTUREWORK about using `eml` in both the `0` and the `1` case?

Reply:
Done in ce66